[Bug] dist/preset.js doesn't exist

### Describe the bug

The [`main` item in `package.json`](https://github.com/storybookjs/addon-svelte-csf/blob/b0010e5d1fb3f0556c04119888d1bf49269a543e/package.json#L14) refers to `dist/preset.js` - but this does not exist, the correct name is `dist/preset/index.js`.

I noticed this in my Svelte/Snowpack based project, since Snowpack was complaining about the missing file during it's dependency building step. I worked around by manually changing the entry to `dist/preset/index.js` (or just `dist/preset`, this works as well) and then Snowpack is happy.

Follow-up: I have found a rather strange issue. At least it seems strange to me, maybe I haven't investigated enough.

* My project uses Snowpack as a dev environment. 
* This was working fine, and then I added Storybook. I set things up so Storybook works now.
* However, Snowpack had issues because it pulls in `addon-svelte-csf` as part of its dependency build step. In encounters the invalid `main` entry in `package.json` - as I reported - and runs into trouble.
* I fixed `package.json` for this addon - again as reported - and now Snowpack was happy.
* However, now Storybook doesn't work anymore! During startup, I see an error that begins like this:

```
ERROR in ./node_modules/@storybook/addon-svelte-csf/dist/parser/svelte-stories-loader.js
Module not found: Error: Can't resolve 'fs' in '/home/sturm/git/myproject/node_modules/@storybook/addon-svelte-csf/dist/parser'
 @ ./node_modules/@storybook/addon-svelte-csf/dist/parser/svelte-stories-loader.js 11:10-23
 @ ./node_modules/@storybook/addon-svelte-csf/dist/preset/index.js
 @ ./src/stories/OliButton.stories.svelte
 @ ./src sync ^\.(?:(?:^|\/|(?:(?:(?!(?:^|\/)\.).)*?)\/)(?!\.)(?=.)[^/]*?\.stories\.svelte)$
 @ ./.storybook/generated-stories-entry.js
...
```

I suspect that I can configure Snowpack to ignore the `.stories.js` files - there's really no reason why they should be included.

However, I thought I should mention the issue I'm seeing. I don't understand why the Storybook startup suddenly has this trouble about resolving `fs` when I change the `main` entry to refer to a **valid** file. Please let me know if I can help with additional information. I'll try to find a workaround now by configuring Snowpack to exclude the unnecessary items.

--- a/package.json
+++ b/package.json
@@ -11,7 +11,8 @@
     "url": "https://github.com/storybookjs/addon-svelte-csf"
   },
   "license": "MIT",
-  "main": "dist/preset.js",
+  "main": "dist/preset",
+  "browser": "dist/index.js",
   "files": [
     "dist/**/*",
     "README.md",

